static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -195,6 +195,10 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 			items["ingestSourceFilesOnly"] = {name : 'Add Files from Server'};
 		}
 
+		if (!isContentRoot && metadata.type === 'Work' && $.inArray('editViewSettings', metadata.permissions) !== -1) {
```

Comment:
I don't think the `!isContentRoot` is necessary here, since a work can't be the contentRoot, although I see that we're doing the same thing in the `if` statement right before this, i'm not sure why.